src/PhpSpreadsheet/Worksheet/ColumnIterator.php

```diff
@@ -186,10 +183,7 @@ public function next()
     public function prev()
     {
         if ($this->position <= $this->startColumn) {
-            throw new PhpSpreadsheetException('Column is already at the beginning of range (' .
-                Cell::stringFromColumnIndex($this->endColumn) . ' - ' .
-                Cell::stringFromColumnIndex($this->endColumn) . ')'
-            );
+            throw new PhpSpreadsheetException('Column is already at the beginning of range (' . Cell::stringFromColumnIndex($this->endColumn) . ' - ' . Cell::stringFromColumnIndex($this->endColumn) . ')');
```

Comment:
@PowerKiKi codeSniffer fails without this change https://travis-ci.org/PHPOffice/PhpSpreadsheet/jobs/228794493